packages/fx-core/src/plugins/resource/appstudio/index.ts

```diff
@@ -51,22 +48,7 @@ export class AppStudioPlugin implements Plugin {
     });
 
     if (stage === Stage.publish) {
-      if (ctx.answers?.platform === Platform.VS) {
-        const appPath = new QTreeNode({
-          type: "folder",
-          name: Constants.PUBLISH_PATH_QUESTION,
-          title: "Please select the folder contains manifest.json and icons",
-          default: `${ctx.root}/.${ConfigFolderName}`,
-        });
-        appStudioQuestions.addChild(appPath);
-
-        const remoteTeamsAppId = new QTreeNode({
-          type: "text",
-          name: Constants.REMOTE_TEAMS_APP_ID,
-          title: "Please input the teams app id in App Studio",
-        });
-        appStudioQuestions.addChild(remoteTeamsAppId);
-      } else if (ctx.answers?.platform === Platform.VSCode) {
+      if (ctx.answers?.platform === Platform.VSCode) {
```

Comment:
Do we need to keep this platform check? Since the following codes don't have this check.
I think it's better to be consistent?
1. Keep the check for other platforms in future.
2. Remove the check if there's only one platform.

Reply:
this check is kept because CLI platform has different behavior for publish command.

Reply:
OK that's no problem.